frontend/src/components/GoalCards/GoalDataController.js

```diff
@@ -72,17 +76,33 @@ function GoalDataController({
     async function fetchGoals(query) {
       setLoading(true);
       try {
+        const mergedGoals = (() => {
+          if (history.location && history.location.state) {
+            return history.location.state.mergedGoals;
+          }
+
+          return null;
+        })();
+
+        let { sortBy } = sortConfig;
+
+        if (mergedGoals) {
+          sortBy = 'mergedGoals';
+        }
+
         const response = await getRecipientGoals(
           recipientId,
           regionId,
-          sortConfig.sortBy,
+          sortBy,
           sortConfig.direction,
           sortConfig.offset,
           goalsPerPage,
           query,
+          mergedGoals || [],
         );
         setData(response);
         setError('');
+        setShouldDisplayMergedSuccess(sortBy === 'mergedGoals' && mergedGoals && mergedGoals.length > 0);
```

Comment:
How does the sortBy relate to a successful merge. Should we track this in its own var?

Reply:
Consult the mockup for details. On a successful merge, the merged goals should display first.

Reply:
Yea I agree but that's not what I'm saying here. We are using a sort order to determine if a sort was successful.